frontend/.env

```diff
@@ -0,0 +1 @@
+REACT_APP_DataBase_link_to_Access_data=http://localhost:4000
```

Comment:
Keep the .env files in gitignore, environment variables should not be updated on a public repository.